### PR TITLE
Added html email verification template

### PR DESCRIPTION
**Pull request by kunaldhand**
Added new html file "email_verification_template.html" for email verification template and edited "validation.py" to make it able to work with new html file template.

### CODE REVIEW

**Comment by kunaldhand** [comment]
I have created a new pull request @edlouth , Kindly revise and merge it. I have also made changes to validation.py so that it can work with new email template.

File: grai-server/app/auth/validation.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def send_validation_email(user: User):
         "token": token,
         "uid": user.pk,
     }
-    email_message = render_to_string("auth/email_verification.txt", c)
+    email_message = render_to_string("auth/email_verification_template.html", c)
```
Comment (edlouth) [inline grai-server/app/auth/validation.py:24]:
```suggestion
    email_message = render_to_string("auth/email_verification.txt", c)
    html_message = render_to_string("auth/email_verification_template.html", c)
```

**Review by edlouth — COMMENTED** [review]
We are so nearly there, just this one bit that needs fixing.

File: grai-server/app/auth/validation.py (hunk 1 above)
Comment (kunaldhand) [reply grai-server/app/auth/validation.py]: Done

**Pull request merged**